File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        low = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[i-1]:
                nums[low]= nums[i]
                low += 1
        return low

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> hashset = new HashSet<>();
        for (int x : nums) hashset.add(x);

        Iterator<Integer> idx = hashset.iterator();
        Integer origin = 0;
        Integer longest = 0;
        while (idx.hasNext()){
            Integer curr = idx.next();
            Integer length = 1;
            if (!hashset.contains(curr - 1)){
                length = 1;
                while (hashset.contains(curr + length)) length += 1;
            }
            longest = Math.max(longest, length);
        }
        return longest;
    }
}
